src/network/asyncore_pollchoose.py

```diff
@@ -75,13 +78,15 @@
 except (ImportError, AttributeError):
     WSAECONNRESET = ECONNRESET
 try:
-    from errno import WSAEADDRINUSE
+    # side-effects on Windows or cruft?
```

Comment:
Yea this is workaround for some Windows crap, it uses winsock which is kind of similar to unix' socket, but it can have its own error numbers, and these aren't defined when running python on other OSes. Maybe there is a nicer workaround but for the time being let's keep it as it is.

Reply:
Updated in next commit...

Reply:
see 533df80